lxd/instance/drivers/driver_qemu.go

```diff
@@ -748,6 +749,16 @@ func (d *qemu) Restart(timeout time.Duration) error {
 	return d.restartCommon(d, timeout)
 }
 
+// Rebuild rebuilds the instance using the supplied image fingerprint as source.
+func (d *qemu) Rebuild(fingerprint string, op *operations.Operation) error {
```

Comment:
Could we have just `Rebuild` and if the fingerprint arg is empty it rebuilds it as empty?